docs/_links.rst

```diff
@@ -16,13 +17,12 @@
 
 .. JSON Schema
 
-.. _Draft 3: https://tools.ietf.org/html/draft-zyp-json-schema-03
-.. _Draft 4: https://tools.ietf.org/html/draft-zyp-json-schema-04
-.. _Draft 5: https://tools.ietf.org/html/draft-wright-json-schema-00
-.. _Draft 6: https://tools.ietf.org/html/draft-wright-json-schema-01
-.. _Draft 7: https://tools.ietf.org/html/draft-handrews-json-schema-01
-
 .. _JSON Schema: https://json-schema.org
+.. _JSON Schema Draft 3: https://tools.ietf.org/html/draft-zyp-json-schema-03
+.. _JSON Schema Draft 4: https://tools.ietf.org/html/draft-zyp-json-schema-04
+.. _JSON Schema Draft 5: https://tools.ietf.org/html/draft-wright-json-schema-00
+.. _JSON Schema Draft 6: https://tools.ietf.org/html/draft-wright-json-schema-01
+.. _JSON Schema Draft 7: https://tools.ietf.org/html/draft-handrews-json-schema-01
```

Comment:
If I understand correctly, only links to Draft 3 and 4 are used. Perhaps we can just delete these other ones? No use in unused code/links.